apps/docs/pages/guides/self-hosting/docker.mdx

```diff
@@ -62,87 +107,94 @@ Update the `.env` file with your own secrets. In particular, these are required:
 
-The services running on the machine are not exposed to the internet by default. We recommend using a reverse proxy such as [NGINX](https://www.nginx.com/) or [Caddy](https://caddyserver.com/) if you want to expose the services publicly:
+## Restarting all services
 
-1. Install Caddy using the official [instructions](https://caddyserver.com/docs/install).
-2. Modify the `Caddyfile` at `./docker/deploy/caddy/Caddyfile`.
-    1. Replace `localhost` with your domain.
-    2. Uncomment the "Dashboard" section if you want to expose the Supabase Dashboard. You **MUST** set a password for this to be secure.
-3. Run `caddy run --config ./docker/deploy/caddy/Caddyfile` to start Caddy.
+You can restart services to pick up any configuration changes by running:
 
-You can now access your services at the following URLs:
+```sh
+docker compose restart
+```
 
-- REST: `https://<your-domain>/proxy/rest/v1/`
-- Auth: `https://<your-domain>/proxy/auth/v1/`
-- Storage: `https://<your-domain>/proxy/storage/v1/`
-- Functions: `https://<your-domain>/proxy/functions/v1/`
-- Realtime: `https://<your-domain>/proxy/realtime/v1/`
-- Dashboard: `https://<your-domain>/` (if you uncommented the Dashboard section in the Caddyfile)
+Be aware that this will result in downtime while the services are restarting.
 
-## Stop 
+## Stopping all services
 
 You can stop Supabase by running `docker compose stop` in same directory as your `docker-compose.yml` file.
 
-## Uninstall
+## Uninstalling
 
-You can stop Supabase by running `docker compose down -v` in same directory as your `docker-compose.yml` file.
+You can stop Supabase by running `docker compose down -v` in same directory as your `docker-compose.yml` file. This will destroy all data in the database and storage volumes, so be careful!
 
-## Advanced configuration
+## Common configuration
 
-Each system can be [configured](../self-hosting#configuration) independently. Some of the most important configuration options are:
+Each system can be [configured](../self-hosting#configuration) independently. Some of the most common configuration options are listed below.
 
-- Consider deploying the database to a different server than the rest of the services
-- Update Storage to use S3 instead of the filesystem backend
-- Configure Auth with a production-ready SMTP server
+### Configuring an email server 
 
-### Setting up Edge Functions
-
-Your Functions are stored in `volumes/functions`. The default setup has a `hello` Function that you can invoke on `http://localhost:8000/functions/v1/hello`. You can add new Functions as `volumes/functions/<Function name>/index.ts`.
+You will need to use a production-ready SMTP server for sending emails. You can configure the SMTP server by updating the the following environment variables:
 
+```sh .env
+SMTP_ADMIN_EMAIL=
+SMTP_HOST=
+SMTP_PORT=
+SMTP_USER=
+SMTP_PASS=
+SMTP_SENDER_NAME=
+```
 
-### Using an external database
```

Comment:
FYI - removing this one because this is the docs for Docker. 

External databases can be covered [separately](https://supabase.com/docs/guides/self-hosting#community)